feat: add server components blog article (#1266)
danielroe/roe.dev: src/content/blog/nuxt-server-components.md

---

> title: 'Nuxt server components: a guide'

Author: ```suggestion
title: 'A guide to Nuxt server components'
```

date: '2023-07-23T14:00:00.000Z'
tags:
  - nuxt
  - server components
description: 
---

'Server components' are a concept you may have come across. They have become increasingly common in the web development ecosystem, for good reason.

It's possible to use server components within [Nuxt](https://nuxt.com/), even if you are generating a static site. That makes it possible to build complex sites that mix dynamic components, server-rendered HTML and even static chunks of markup.

> :span{class="i-ri:lightbulb-fill h-4 w-4"} Believe it or not, Nuxt has had server components since before React.

> ## Some key terms

Author: ```suggestion
## Similar but different
```

> There are a few terms floating around that are worth mentioning:

Author: ```suggestion
There are a other similar-sounding terms floating around that are worth mentioning:
```
